pkg/apis/openstackproviderconfig/v1alpha1/types.go

```diff
@@ -81,6 +80,37 @@ type OpenstackProviderSpec struct {
 	ConfigDrive *bool `json:"configDrive,omitempty"`
 }
 
+type Tags struct {
+	// Tags added to all resources created by cluster
+	// Default tags include: cluster-api-provider-openstack, clusterName
+	Cluster ClusterTags `json:"cluster,omitempty"`
+
+	// Server tags: tags servers only
+	Server ServerTags `json:"server,omitempty"`
+
+	// Port tags: tags only applied to networking resources created
```

Comment:
Port tags seems confusing... only ports or other stuffs related to network(network, secgroup) will be taged?

Reply:
That sould say network tags, and yeah, it is only tagging resources created by running cluster-api-provider-openstack